testing/README.md

```diff
@@ -0,0 +1,219 @@
+# Implementation Report
+
+This directory contains the
+[WoT Thing Description Implementation report](https://w3c.github.io/wot-thing-description/testing/report.html)
```

Comment:
The documet needs to be updated for the profile. Links are still pointing to the TD.

Reply:
Actually, I updated this entire README at some point but it seems it did not get checked in, and now I can't find the version I edited.  The instructions here are now completely wrong.  Anyway, I edited the whole thing... again.